spec/paper_trail/model_spec.rb

```diff
@@ -486,249 +468,251 @@
   end
 
   context "A subclass" do
+    let(:foo) { FooWidget.create }
```

Comment:
Nit: What do you think of using `let!` everywhere or at least when we know it'll be called?

Reply:
I don't have strong opinions. I'd rather have any test, however written (within reason) than no test. 

Using `let!` when not needed makes tests slower. Using either form of `let`, compared to local variables, makes tests harder to understand, IMO.

My personal goal is to see the whole test on one page, so that I can understand it quickly. So, I tend to prefer local variables over either form of `let`. This leads to duplication between tests, but I prefer some duplication if it makes tests easier to understand.

Reply:
Sounds good!  I agree, it can be difficult to scroll up a large file to see where variables are defined 👍